src/main/java/net/dv8tion/jda/internal/handle/GuildUpdateHandler.java

```diff
@@ -81,7 +81,7 @@ protected Long handleInternally(DataObject content)
         Guild.NSFWLevel nsfwLevel = Guild.NSFWLevel.fromKey(content.getInt("nsfw_level", -1));
         Guild.ExplicitContentLevel explicitContentLevel = Guild.ExplicitContentLevel.fromKey(content.getInt("explicit_content_filter"));
         Guild.Timeout afkTimeout = Guild.Timeout.fromKey(content.getInt("afk_timeout"));
-        Locale locale = Locale.forLanguageTag(content.getString("preferred_locale", "en-US"));
```

Comment:
Looks like we were defaulting it to `en-US` here, so maybe it was just an erroneous default in the EntityBuilder? 😬